open_api_spec.yml

```diff
@@ -5416,7 +5422,38 @@ paths:
                     headers: *req-id-header
                 default:
                     $ref: "#/components/responses/Error"
-
+    /management/audit_logs:
+        get:
+            summary: Get audit logs
+            description: Get audit logs
+            operationId: getAudits
+            tags:
+                - Workspace Management
+            parameters:
+                - in: query
+                  name: timePeriod
+                  required: false
+                  description: The final time period to fetch audit logs (day/week)
+                  schema:
+                      type: string
+                      enum:
+                      - DAY
+                      - WEEK
+                - in: query
+                  name: cursor
+                  required: false
+                  description: The next ID to start fetching audit logs from
+                  schema:
+                    type: string
+            responses:
+                '200':
+                    description: Audit logs from the requested time period onward
+                    headers:
+                        X-Request-ID:
+                        $ref: '#/components/headers/X-Request-ID'
+                    default:
```

Comment:
'default' should go 1 tab back so it will be on the same level as '200' (its a different response)